exercises/dnd-character/mix.exs

```diff
@@ -0,0 +1,25 @@
+defmodule DndCharacter.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :dnd_character,
+      version: "1.1.0",
+      # elixir: "~> 1.8",
+      start_permanent: Mix.env() == :prod,
+      deps: deps()
+    ]
+  end
+
+  # Run "mix help compile.app" to learn about applications.
+  def application do
+    [
+      extra_applications: [:logger]
+    ]
+  end
+
+  # Run "mix help deps" to learn about dependencies.
+  defp deps do
+    []
```

Comment:
I'd like to keep this file as "stock" as possible. Could we replace this line with:
```
    [
      # {:dep_from_hexpm, "~> 0.3.0"},
      # {:dep_from_git, git: "https://github.com/elixir-lang/my_dep.git", tag: "0.1.0"}
    ]
```